tasks/setup-RedHat.yml

```
@@ -10,3 +10,10 @@
     name: "{{ postgresql_python_library }}"
     state: present
     enablerepo: "{{ postgresql_enablerepo | default(omit, true) }}"
+
+- name: Ensure PostgreSQL systemd have the rigth datadir
```

spelling error "right"
```suggestion
- name: Ensure PostgreSQL systemd have the right datadir
```